src/index.ts

```diff
@@ -192,7 +203,7 @@ export default class HDKeyring implements Keyring<SerializedHDKeyring> {
     const newPath = `${index}`
 
     const childNode = this.#hdNode.derivePath(newPath)
-    const wallet = new Wallet(childNode)
+    const wallet = new Wallet(childNode.privateKey)
```

Comment:
Was this not working before? If so, we should add tests that show it failing to work, as well.

Reply:
No, there's an issue in ethers that will be fixed in the future.

Without this, ethers throws an error here: https://github.com/ethers-io/ethers.js/blob/73a46efea32c3f9a4833ed77896a216e3d3752a0/packages/wallet/src.ts/index.ts#L66

I have a PR open for ethers, but the maintainer recommended this change for now to continue working and I have not seen any issues from it.

Open to recommendations on tests for this one.